app/Console/Commands/MigrateCommand.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * MigrateCommand.php
+ *
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+namespace App\Console\Commands;
+
+use App\Console\LnmsCommand;
+use Illuminate\Contracts\Cache\LockTimeoutException;
+use Illuminate\Support\Facades\Cache;
+use Log;
+use Schema;
+
+class MigrateCommand extends LnmsCommand
+{
+    protected $signature = 'migrate:librenms {--force : Force running even if already running}';
```

Comment:
Didn't manage to override the real command, so gave it a "work in progress" name.

Reply:
@murrant can probably solve it. But come thinking about it, wondering if this just complicates stuff for no reason?

Reply:
Wouldn't schema:migrate be more appropriate.

"THE INTERNET" indicates setting $name to the command name will work, but I have no idea.

Reply:
overriding the command "name" itself works, but not extending it or trying to call the original one.
Think its due to how its created;

https://github.com/laravel/framework/blob/ccefc764871fe263dd20f8f7108c18180f78241b/src/Illuminate/Database/MigrationServiceProvider.php#L117-L122